Sources/Impl/PinLayoutImpl+Coordinates.swift

```diff
@@ -24,6 +24,19 @@ extension PinLayoutImpl {
     internal func top(_ context: Context) {
         setTop(0, context)
     }
+
+    @discardableResult
+    internal func top(_ value: CGFloat, _ context: Context) -> PinLayout {
+        setTop(value, context)
```

Comment:
You don't need to add this method if you call `setTop(value, context), setBottom(value, context) ...` in `func all(_ value: CGFloat)`

Reply:
Ah, ok you can keep it that way, I use this pattern somewhere else 😊

Reply:
:)

Reply:
But could you update the method `func top(_ value: CGFloat) -> PinLayout` in PinLayoutImpl.swift to:
```
@discardableResult
    func top(_ value: CGFloat) -> PinLayout {
        return top(value, { return "top(\(value))" })
    }
```